docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md

```diff
@@ -56,22 +58,47 @@ The **Backups** page allows you to create new backups that start immediately, as
 
+<!-- 
+
 {{< note title="Note" >}}
 
 Versions of YugabyteDB Anywhere prior to 2.11.2.0 do not support backups of YSQL databases that use `enum` types. To mitigate the issue, it is recommended that you use the `ysql_dump` utility in combination with the `/COPY` action as a workaround.
 
 {{< /note >}}
 
+-->
+
 <!-- The preceding note should say 2.11.2.0. Careful with search and replace on version numbers! -->
 
 For information on how to schedule backups for a later time or as a recurring task, see [Schedule universe YSQL data backups](../../schedule-data-backups/ysql/).
 
-To view detailed information about an existing backup, click on it to open **Backup Details**. In addition to actions such as deleting and restoring the backup, as well as restoring and copying the database location, you can use **Backup Details** to add an incremental backup for universes that had the YB Controller automatically installed during their creation. The same universes allow you to configure their throttle parameters by clicking **... > Configure Throttle Parameters**, as per the following illustration:
+To view detailed information about an existing backup, click on it to open **Backup Details**. In addition to actions such as deleting and restoring the backup, as well as restoring and copying the database location, you can use **Backup Details** to add an [incremental backup](#create-incremental-backup) for universes that had the YB Controller automatically installed during their creation. The same universes allow you to configure their throttle parameters by clicking **... > Configure Throttle Parameters**, as per the following illustration:
+
+![Throttle parameters](/images/yp/backup-throttle-config-button.png)
+
+You define throttle parameters to enhance your universe's backups and restore performance using the **Configure Throttle Parameters** dialog shown in the following illustration:
+
+![Throttle](/images/yp/backup-restore-throttle.png)
+
+To access a list of all backups from all universes, including the deleted universes, navigate to **Backups** on the YugabyteDB Anywhere left-side menu, as per the following illustration:
+
+![Backups](/images/yp/backups-list.png)
+
+## Create incremental backups
+
+Incremental backups are a way of taking differential backups on top of a complete backup. To reduce the length of time spent on each backup, only SST files that are new to YugabyteDB and not present in the previous backups are incrementally backed up. For example, in most cases, for incremental backups occurring every hour, the 1-hour delta would be significantly smaller compared to the complete backup. The restore happens until the point of the defined increment.
```

Comment:
Lets not use the word 'differential' anywhere as differential backups is another term and means something slightly different.

Reply:
Removed "differential"